src/helpers/expectNonZeroAddressProvider.js

```diff
@@ -0,0 +1,31 @@
+export default function expectNonZeroAddressProvider (web3) {
+  function isChecksumAddress (address) {
+    address = address.replace('0x', '')
+    var addressHash = web3.sha3(address.toLowerCase())
+    for (var i = 0; i < 40; i++) {
+      if (
```

Comment:
@mikec not sure I understand what's going on here? When testing with legitimate addresses, I'm getting a failure here.